LayoutTests/http/wpt/html/dom/scroll-to-text-fragment/no-iframe-match.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html><!-- webkit-test-runner [ ScrollToTextFragmentIndicatorEnabled=false ] -->
```

Comment:
do WPTs usually have webkit-test-runner headers in them? how does that work with the upstreaming??

Reply:
(I admit I know nothing about WPT so take this feedback with a grain of salt)

Reply:
My other tests have them because of the indicator flash. I have not uploaded any of these tests yet, but I think it's important to have a test. It's very possible that this and the other tests in this folder will be combined/replaced with the existing tests.

Reply:
Tests under http/wpt are NOT shared with other browsers. The ones under imported/w3c/web-platform-tests are.

Reply:
(Megan agrees this is wrong and that these tests probably need to move out of WPTs, but is going to do that as a unit, instead of just for this one).